.travis.yml

```diff
@@ -19,4 +19,5 @@ script:
   - python dbconnector.py
   # run tests
   - python -m pytest test/unit/test.py
```

Comment:
@vlad-isayko you advice doesn't work on my system. Does it work on yours?

Reply:
Yes, there is a problem with the name of the test files, they must begin with 'test_' in order for pytest to be able to find them.

Reply:
@vlad-isayko thanks for the confirmation of my fears. I am hesitant to rename the tests, because @EmbeddAlex who wrote them obviously doesn't have the problem with running them, and renaming will likely break his setup. There is no instructions how to run the tests in `README.md` either.

Reply:
I think that this can be left unchanged in this PR.